### PR TITLE
Only show the promulgated charms if they exist

### PR DESCRIPTION
**Pull request by hatched**
In an effort to match the search results from jujucharms.com this branch will only show the promulgated charms/bundles if they eixist, and if not will fall back to the community results.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Search for:
- parse-server
  - You should  only get the single Community result
- apache
  - You should get a long list of values
- apache2
  - You should only see two results

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5505/
Test PASSed.

**Comment by jrwren** [comment]
👍 NO QA

File: jujugui/static/gui/src/app/assets/javascripts/jujulib/charmstore.js [hunk 1]
```diff
@@ -319,17 +319,20 @@ var module = module;
...
+        'include=charm-metadata',
+        'include=charm-config',
+        'include=supported-series',
+        'include=bundle-metadata',
+        'include=extra-info',
+        'include=tags',
+        'include=owner',
+        'include=stats'
+      ];
+      var path = this._generatePath('search', includes.join('&'));
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/javascripts/jujulib/charmstore.js]: Thanks for cleaning this up!

**Comment by huwshimi** [comment]
Searching for "parse-server" returned 0 results, but the other two searches worked fine.

**Comment by hatched** [comment]
@huwshimi could you try with production instead of staging/guimaas charmstore - I think only production has parse-server

**Comment by huwshimi** [comment]
QA OK :+1: using production.

**Comment by hatched** [comment]
Thanks @jrwren @huwshimi 

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**